recipes-support/rauc-hawkbit-updater/rauc-hawkbit-updater_git.bb

```
@@ -0,0 +1,26 @@
+SUMMARY = "The RAUC hawkBit updater operates as an interface between the RAUC D-Bus API and the hawkBit DDI API."
+HOMEPAGE = "https://github.com/rauc/rauc-hawkbit-updater"
+
+LICENSE = "LGPLv2.1"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=1a6d268fd218675ffea8be556788b780"
+
+SRC_URI = "git://github.com/rauc/rauc-hawkbit-updater.git;protocol=https"
+SRCREV = "84da0e26e5adff7000b60fe22e821570108286a4"
+S = "${WORKDIR}/git"
+
+inherit cmake pkgconfig systemd
+
+SYSTEMD_SERVICE_${PN} = "rauc-hawkbit-updater.service"
+
+EXTRA_OECMAKE_append = " -DWITH_SYSTEMD=ON"
+
+DEPENDS = "systemd curl glib-2.0-native json-glib"
```

I'd prefer not depending hard on systemd here, especially as rauc-hawkbit-updater explicitly provdes a CMake switch for this.

Thus we can simply make use of the CMake option `WITH_SYSTEMD` and introduce a PACKAGECONFIG for this
The rest we can make depending on `DISTRO_FEATURE` `systemd`:

```suggestion
SYSTEMD_SERVICE_${PN} = "${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'rauc-hawkbit-updater.service', '', d)}"

PACKAGECONFIG ??= "\
    ${@bb.utils.filter('DISTRO_FEATURES', 'systemd', d)} \
    "
PACKAGECONFIG[systemd] = "-DWITH_SYSTEMD=ON,,systemd"

DEPENDS = "curl glib-2.0-native json-glib"
```

---

Good point! I will change this.